### PR TITLE
second exercises

### PR DESCRIPTION
**Pull request by clever-cr**

### CODE REVIEW

File: reverseVowels.js [hunk 1]
```diff
@@ -0,0 +1,6 @@
+function reverseVowels(str) {
+    let vowels = ["a", "e", "i", "o", "u", "A", "E", "O", "I", "U"]
```
Comment (IbrahimBagalwa) [inline reverseVowels.js]: It could be fine if you turn the `let vowels` into a constant because it will never change

File: reverseVowels.js [hunk 2]
```diff
@@ -0,0 +1,6 @@
...
+    let onlyVowels = str.split("").filter((letter) => vowels.includes(letter))
```
Comment (IbrahimBagalwa) [inline reverseVowels.js]: Here as well

File: phoneNumber.js [hunk 3]
```diff
@@ -0,0 +1,5 @@
+function createPhoneNumber(numbers) {
+    let stringNumbers = numbers.join("").toString()
```
Comment (musayann) [inline phoneNumber.js]: Should be a constant since it wasn't re-assigned

Comment (IbrahimBagalwa) [inline phoneNumber.js]: It could be fine if you make `stringNumbers` a constant

Comment (IbrahimBagalwa) [inline phoneNumber.js]: I think there is not need to stringify the `stringNumbers` because when you joined it became a string

**Review by IbrahimBagalwa — COMMENTED** [review]
Great job @clever-cr 
I left comment to improve the code base

**Pull request merged**